apps/settings/js/panels/feedback_send/feedback_send.js

```diff
@@ -106,13 +110,14 @@ define(function(require) {
         currentSetting['deviceinfo.hardware'];
       this._sendData.locale =
         currentSetting['language.current'];
+      this._sendData.happy = this._isHappy();
```

Comment:
Hmmm, I think we don't have to keep `happy` field in asyncStorage because every time users jump into this panel, this value will keep changing based on what they choose.